app/models/organization.rb

```diff
@@ -11,6 +11,17 @@ class Organization < ApplicationRecord
 
   acts_as_followable
 
+  before_validation :downcase_slug
```

Comment:
We have a rubocop that forces us to put callbacks in a certain order. Since I had to move the before_destroy callback before the `dependent: nil` statements the rest of these had to come with it.